arangod/Network/ConnectionPool.cpp

```diff
@@ -177,7 +177,8 @@ ConnectionPool::Context::Context(std::shared_ptr<fuerte::Connection> c,
     : fuerte(std::move(c)), lastLeased(t), leases(l) {}
 
 std::shared_ptr<fuerte::Connection> ConnectionPool::createConnection(fuerte::ConnectionBuilder& builder) {
-  auto idle = std::chrono::milliseconds(_config.idleConnectionMilli);
+  //auto idle = std::chrono::milliseconds(_config.idleConnectionMilli);
+  auto idle = std::chrono::milliseconds(2);
```

Comment:
Fine for testing, but we need remember to revert this change before merging.

Reply:
Indeed!